src/java_bytecode/java_types.h

```diff
@@ -374,4 +374,45 @@ void get_dependencies_from_generic_parameters(
   const typet &,
   std::set<irep_idt> &);
 
+class java_specialised_generic_class_typet : public java_generic_class_typet
```

Comment:
The coding guidelines ask for American spelling for identifiers. The same applies to virtually any other hunk in the diff ;)